man/rgbasm.5

```diff
@@ -324,9 +341,33 @@ with a zero constant as either operand will be constant 0, and
 .Sq ||
 with a non-zero constant as either operand will be constant 1, even if the other operand is non-constant.
 .Pp
-.Ic \&!
+.Sq \&!
```

Comment:
```suggestion
.Ic \&!
```

is consistent with the other definitions above. I think we should stick to that for this PR.

Reply:
...No it's not? I see `.Sq &&`, `.Sq &`, and `.Sq ||` above.

Reply:
One `.Ic ~` on line 300, one `.Ic %` on line 303. Agreed we can fix/discuss those separately. To avoid blocking this PR, I'm leaving this change alone, making the `.Sq \&!` consistent with the other operators whose constant-ness is being discussed right in this "paragraph", instead of undoing a change only to redo it in another PR along with many others.